content/_includes/utility.macros.njk

```diff
@@ -219,7 +219,7 @@ params:
 
   <time
     datetime="{{ start | getDate('iso') }}"
-    {{ attr_if('pubdate', not not is_pubdate) }}>
+    {{ attr_if('pubdate', is_pubdate) }}>
```

Comment:
@mirisuzanne I couldn't find any reason to double-coerce this value? It should always be a boolean to begin with...